package.json

```diff
@@ -40,27 +41,28 @@
     "html-webpack-plugin": "2.29.0",
     "i18next": "^10.3.0",
     "jest": "20.0.4",
+    "material-design-icons-iconfont": "^4.0.5",
     "material-ui": "^0.20.0",
     "object-assign": "4.1.1",
     "postcss-flexbugs-fixes": "3.2.0",
     "postcss-loader": "2.0.8",
     "promise": "8.0.1",
     "raf": "3.4.0",
-    "react": "^16.2.0",
+    "react": "^16.3.0",
```

Comment:
Did you try the latest version and it didn't work? Or what's the reason for not upgrading react to the latest version?

Reply:
Yeah I got some issues with d2-ui, so I bumped it to the one I needed that didn't have any problems.

This is going to go away eventually once we start porting apps to the app shell. We'll have a common react version and move everything to the newest.